frontend/src/widgets/TotalHrsAndGranteeGraph.js

```diff
@@ -152,6 +131,7 @@ export function TotalHrsAndGranteeGraph({ data, dateTime }) {
         showgrid: false,
         b: 0,
         t: 0,
+        autotypenumbers: 'strict',
```

Comment:
Odd that you would need to add this the documentation says this is the default value.